data/database.sql

```diff
@@ -363,7 +363,7 @@ CREATE TABLE IF NOT EXISTS `site_scan_history` (
   CONSTRAINT `fk_site_scan_history_sites1`
     FOREIGN KEY (`sites_id`)
     REFERENCES `sites` (`id`)
-    ON DELETE NO ACTION
+    ON DELETE CASCADE
```

Comment:
Correct. But the foreign keys in the database are there to protect the database integrity even when the application doesn't (such as when someone directly accesses the db). I was just curious if you had a specific use case that mean that the keys shouldn't change together, but it doesn't seem like you do.